### PR TITLE
Add OData View Test

### PR DESCRIPTION
**Pull request by madhawa-gunasekara**

### CODE REVIEW

File: integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/odata/ODataViewTestCase.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+		List<File> sqlFileLis = new ArrayList<>();
+		sqlFileLis.add(selectSqlFile("CreateODataTables.sql"));
+		sqlFileLis.add(selectSqlFile("Customers.sql"));
+		deployService(serviceName,
+		              createArtifact(getResourceLocation() + File.separator + "dbs" + File.separator + "odata" +
+		                             File.separator + "ODataSampleSuperTenantService.dbs", sqlFileLis));
+		webAppUrl = dssContext.getContextUrls().getWebAppURL();
+	}
+
+    @AfterClass(alwaysRun = true)
```
Comment (vinok88) [inline integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/odata/ODataViewTestCase.java]: Formatting issue?

Comment (madhawa-gunasekara) [reply integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/odata/ODataViewTestCase.java]: :+1:  Fixed

File: integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/odata/ODataViewTestCase.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        Assert.assertEquals(response[0], ODataTestUtils.OK);
+        endpoint = webAppUrl + "/odata/" + serviceName + "/" + configId + "/";
+        response = ODataTestUtils.sendGET(endpoint, headers);
+        Assert.assertEquals(response[0], ODataTestUtils.OK);
+        Assert.assertTrue(response[1].toString().contains("USA"));
+        Assert.assertTrue(!response[1].toString().contains("France"));
+        Assert.assertTrue(!response[1].toString().contains("Finland"));
+    }
+
+}
```
Comment (bsenduran) [inline integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/odata/ODataViewTestCase.java]: Better to end with a empty newline

Comment (madhawa-gunasekara) [reply integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/odata/ODataViewTestCase.java]: :+1: done

File: integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/odata/ODataViewTestCase.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    @Test(groups = {"wso2.dss"}, description = "OData View Test")
+    public void validateViewTestCase() throws Exception {
+        String endpoint = webAppUrl + "/odata/" + serviceName + "/" + configId + "/" + viewName;
+        Map<String, String> headers = new HashMap<>();
+        headers.put("Accept", "application/json");
+        Object[] response = ODataTestUtils.sendGET(endpoint, headers);
+        Assert.assertEquals(response[0], ODataTestUtils.OK);
+        endpoint = webAppUrl + "/odata/" + serviceName + "/" + configId + "/";
+        response = ODataTestUtils.sendGET(endpoint, headers);
+        Assert.assertEquals(response[0], ODataTestUtils.OK);
```
Comment (chanikag) [inline integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/odata/ODataViewTestCase.java]: Add assert messages?

**Pull request merged**